hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/TestUtils.java

```diff
@@ -475,7 +475,6 @@ public static void quasiCloseContainer(ContainerManagerV2 containerManager,
   public static StorageContainerManager getScmSimple(OzoneConfiguration conf)
       throws IOException, AuthenticationException {
     SCMConfigurator configurator = new SCMConfigurator();
-    configurator.setSCMHAManager(MockSCMHAManager.getInstance(true));
```

Comment:
You will need this line: 
`conf.setBoolean(ScmConfigKeys.OZONE_SCM_HA_ENABLE_KEY, true);`

`SCMManagerImpl` will return isLeader immediately without enabling this HA:

```
    if (!SCMHAUtils.isSCMHAEnabled(conf)) {
      // When SCM HA is not enabled, the current SCM is always the leader.
      return Optional.of((long)0);
    }
```

Reply:
So currently the `cluster.waitForClusterToBeReady` returns quickly even the SCM has not become leader yet. 

I am guessing CI can still pass become you also have changed the rpc timeout parameters. So either the cluster wait for ready or the parameter change can solve the `not leader problem`. 

I am not if we need both. Maybe we only need to keep `cluster.waitForClusterToBeReady`?

Reply:
Good catch!